frontend/src/app/organization/organization-admin/organization-admin.component.ts

```diff
@@ -1,10 +1,21 @@
-import { Component } from '@angular/core';
-import { Router } from '@angular/router';
-import { permissionGuard } from 'src/app/permission.guard';
+/**
+ * The Admin Organization List page retrieves and displays a list of
+ * CS organizations and provides functionality to create/delete them.
+ *
+ * @author Ajay Gandecha, Jade Keegan, Brianna Ta, Audrey Toney
+ * @copyright 2024
+ * @license MIT
+ */
+
+import { Component, Signal } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
 import { Organization } from '../organization.model';
 import { MatSnackBar } from '@angular/material/snack-bar';
-import { OrganizationAdminService } from './organization-admin.service';
 import { Observable } from 'rxjs';
+import { Profile } from '/workspace/frontend/src/app/profile/profile.service';
```

Comment:
So I am having some problems with getting the relative paths to be recognized - I am going to try to restart my DevContainer and see if it helps - yeah `/workspace` included here was automatically generated and seems odd since I do not believe this has happened before.

Reply:
Yeah @KrisJordan very strange.. It seems that files outside of `../` are not being recognized. There does not seem to be a change in any of the config files, so I am not sure what the cause might be. Thoughts? Am still looking into, rebuilding the container did not help.

Reply:
Update: Ah, it seems using `'../../profile/profile.service'` works, using `../../../` chaining. Not sure if this is what we want lol